lib/utils/array.js

```diff
@@ -1,3 +1,15 @@
-export default function last(arr) {
+export function last(arr) {
   return arr[arr.length - 1];
 }
+
+export const arrBuf2Base64 = buffer => {
+  let base64String = null;
+  try {
+    base64String = Buffer.from(buffer).toString('base64');
+  } catch (err) {
+    //eslint-disable-next-line no-console
+    console.error('Could not use the file specified:', err);
```

Comment:
Should we really catch this error here? Isn't it better to re-throw it and handle it in the caller to display the error to the user? 🤔

Reply:
Ah yea you have a point. I could just have the getBase64FileData function use the try catch and resolve with null if it catches an error. Taking it a bit further, does it make sense anymore to wrap `Buffer.from(buffer).toString('base64')` in arrBuf2Base64 anymore? It has turned into a one-liner thing now so technically the `getBase64FileData` can import the Buffer class and perform the one-liner. Thoughts?

Reply:
Hmm this should never throw anyways, if you send a buffer or string to `Buffer.from` it won't throw (afaik), so maybe it's not even needed to try/catch it if you know you got a valid buffer/string?